### PR TITLE
switch to `@napi-rs/canvas` 

### PR DESCRIPTION
**Pull request by SuperchupuDev**
i hate node-canvas i hate node-canvas

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
     },
     "homepage": "https://github.com/Fluorinebot/fluorine/#readme",
     "dependencies": {
+        "@napi-rs/canvas": "^0.1.27",
```
Comment (xiboon) [inline package.json:38]: bad

File: src/commands/profile/view.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import type FluorineClient from '#classes/Client';
 import { fragmentText } from '#util/fragmentText';
-import canvas from 'canvas';
+import canvas, { GlobalFonts } from '@napi-rs/canvas';
```
Comment (xiboon) [inline src/commands/profile/view.ts:3]: lol so shitty

File: src/util/fragmentText.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import type { CanvasRenderingContext2D } from 'canvas';
+import type { SKRSContext2D } from '@napi-rs/canvas';
```
Comment (xiboon) [inline src/util/fragmentText.ts:1]: bro really basd names skrws context 2d ????? its not the 90s anymore use 3d

File: src/commands/profile/view.ts [hunk 4]
```diff
@@ -83,7 +74,7 @@ export async function run(client: FluorineClient, interaction: ChatInputCommandI
     ctx.clip();
     ctx.drawImage(avatar, 30, 7, 110, 110);
 
-    const attachment = new AttachmentBuilder(canva.toBuffer(), { name: 'profile.png' });
+    const attachment = new AttachmentBuilder(canva.toBuffer('image/png'), { name: 'profile.png' });
```
Comment (xiboon) [inline src/commands/profile/view.ts:77]: imagine  needing to type 10 more letterS SMH

File: src/commands/profile/view.ts [hunk 5]
```diff
@@ -17,18 +17,9 @@ export async function run(client: FluorineClient, interaction: ChatInputCommandI
...
-        family: 'Poppins',
-        weight: 'normal'
-    });
-    canvas.registerFont(`${__dirname}/../../../assets/Poppins-Medium.ttf`, {
-        family: 'Poppins',
-        weight: 'bold'
-    });
+    GlobalFonts.registerFromPath(`${__dirname}/../../../assets/Inter-Light.ttf`);
+    GlobalFonts.registerFromPath(`${__dirname}/../../../assets/Poppins-Regular.ttf`);
+    GlobalFonts.registerFromPath(`${__dirname}/../../../assets/Poppins-Medium.ttf`);
```
Comment (xiboon) [inline src/commands/profile/view.ts:22]: isnt it shitting

**Review by hkubus — DISMISSED** [review]
bad

**Comment by xiboon** [comment]
SHUT UP STUPID

**Comment by xiboon** [comment]
BITCH\

**Comment by xiboon** [comment]
FUCKING

**Review by Nyameliaaaa — APPROVED** [review]
i havent read the code idk

**Comment by SuperchupuDev** [comment]
thanks

**Pull request merged**